docs/reference/indices/templates.asciidoc

```diff
@@ -1,9 +1,21 @@
-[[indices-templates]]
+[[indices-templates-v1]]
 === Put index template API
 ++++
 <titleabbrev>Put index template</titleabbrev>
 ++++
 
+This documentation is about V1 (version 1) index templates, are deprecated and will be replaced by
+V2 templates. For V2 templates please see the <<indices-templates-v1,V1 template documentation>>.
+
+[NOTE]
+====
+In {es} 7.8 the `prefer_v2_templates` querystring parameter was introduced to the index, update,
+bulk, and create index APIs to allow configuring whether to favor V2 or V1 index templates. In 7.x
+the default value for this parameter is `false` meaning that the V1 index templates take precedence.
+In 8.x the default value for the parameter is `true` meaning that <<indices-templates,V2 index
```

Comment:
This xref is causing the build failure. You either need to change the anchor in index-templates.asciidoc to [[indices-templates]], or update this xref. 

I think I'd stick with the indices-templates anchor that was used for v1, as there are likely external links to the topic. I know the plural indices is weird, but that pattern is consistent through the rest of the index APIs.

Reply:
Sounds good, thanks for taking a look @debadair!